dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4501,6 +4501,59 @@ paths:
                   $ref: '#/components/schemas/Workflow'
                 type: array
           description: default response
+  /users/{userId}/services/refresh:
+    post:
+      operationId: syncUserServices
+      parameters:
+      - in: path
+        name: userId
+        required: true
+        schema:
+          format: int64
+          type: integer
+      requestBody:
```

Comment:
there's a body or are all post endpoints like this for some reason?

Reply:
Hmm, looks like OpenAPI3 POST endpoints are like this for some reason. Possibly a code generator issue?